component/Page/BaseContainer.js

```diff
@@ -8,7 +8,7 @@ import React from "react";
  * @returns {React.ReactNode}
  */
 export function BaseContainer({ children, className = '' }) {
-    return <div className={`flex flex-col content-center sm:w-full items-center justify-center h-screen ${className}`}>
+    return <div className={`flex flex-col content-center sm:w-full items-center justify-center h-[calc(100dvh)] ${className}`}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The height of the container is set to `100dvh` which seems to be a typo. The correct unit for viewport height is `vh`, not `dvh`. Please correct this.
```diff
- return <div className={`flex flex-col content-center sm:w-full items-center justify-center h-[calc(100dvh)] ${className}`}>
+ return <div className={`flex flex-col content-center sm:w-full items-center justify-center h-[calc(100vh)] ${className}`}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->